plugins/modules/lambda_layer.py

```diff
@@ -0,0 +1,355 @@
+      - x86_64
+
+# Create a layer version from a zip in the local filesystem
+- name: Create a new layer from a zip in the local filesystem
+  lambda_layer:
+    state: present
+    name: sample-layer
+    description: 'My Python layer'
+    content:
+      zip_file: 'python_layer.zip'
+    compatible_runtimes:
+      - python3.6
+      - python3.7
+    license_info: MIT
+    compatible_architectures:
+      - x86_64
+
+# Delete a layer version
+- name: Delete a layer version
+  lambda_layer:
+    state: present
+    name: sample-layer
+    version: 2
+'''
+
+RETURN = '''
+layer_version:
+  description: info about the layer version that was created or deleted.
+  returned: always
+  type: complex
+  contains:
+    content:
+        description: Details about the layer version.
+        returned: I(state=present)
+        type: complex
+        contains:
+          location:
+            description: A link to the layer archive in Amazon S3 that is valid for 10 minutes.
+            returned: I(state=present)
+            type: str
+            sample: "https://awslambda-us-east-1-layers.s3.us-east-1.amazonaws.com/snapshots/123456789012/pylayer-9da91deffd3b4941b8baeeae5daeffe4"
+          code_sha256:
+            description: The SHA-256 hash of the layer archive.
+            returned: I(state=present)
+            type: str
+            sample: "VLluleJZ3HTwDrdYolSMrS+8iPwEkcoXXaegjXf+dmc="
+          code_size:
+            description: The size of the layer archive in bytes.
+            returned: I(state=present)
+            type: int
+            sample: 9473675
+          signing_profile_version_arn:
+            description: The Amazon Resource Name (ARN) for a signing profile version.
+            returned: When a signing profile is defined
+            type: str
+          signing_job_arn:
+            description: The Amazon Resource Name (ARN) of a signing job.
+            returned: When a signing profile is defined
+            type: str
+    layer_arn:
+        description: The ARN of the layer.
+        returned: if the layer version exists or has been created
+        type: str
+        sample: "arn:aws:lambda:eu-west-2:123456789012:layer:pylayer"
+    layer_version_arn:
+        description: The ARN of the layer version.
+        returned: if the layer version exists or has been created
+        type: str
+        sample: "arn:aws:lambda:eu-west-2:123456789012:layer:pylayer:2"
+    description:
+        description: The description of the version.
+        returned: I(state=present)
+        type: str
+    created_date:
+        description: The date that the layer version was created, in ISO-8601 format (YYYY-MM-DDThh:mm:ss.sTZD).
+        returned: if the layer version exists or has been created
+        type: str
+        sample: "2022-09-28T14:27:35.866+0000"
+    version:
+        description: The version number.
+        returned: if the layer version exists or has been created
+        type: str
+        sample: 1
```

Comment:
Is it returned as str or int? 
```suggestion
        sample: "1"
```